tmail-backend/pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>jmap/extensions-cassandra</module>
         <module>jmap/extensions-es-v7</module>
 
+        <module>rate-limit</module>
```

Comment:
<module>rate-limit</module>

=> 

<module>rate-limit/rate-limit-api</module>

(We will need multi-modules EG with Cassandra implementation)